sdk/communication/Azure.Communication.CallingServer/src/Models/AnswerCallResult.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary> The result from creating a call. </summary>
+    public class AnswerCallResult
+    {
+        internal AnswerCallResult(CallConnection callConnection, CallConnectionProperties callProperties)
```

Comment:
Why is the type called CallConnectionProperties, but the variable name callProperties?

Reply:
I would tend to agree here but the type has a "long name" and the rule of thumb is that input parameter names match their type names, aside from primitive types of course. So maybe consider changing `CallConnectionProperties` to `CallProperties` with a matching `callProperties` parameter name or make it match the other way.